core-bundle/src/OptIn/OptIn.php

```diff
@@ -91,8 +91,6 @@ public function purgeTokens(): void
                 foreach ($related as $table => $id) {
                     /** @var class-string<Model> $class */
                     $class = $adapter->getClassFromTable($table);
-
-                    /** @var Model $model */
```

Comment:
<img width="543" alt="" src="https://user-images.githubusercontent.com/1192057/143822154-fae5b0f1-6a6d-4e3a-aa69-026bd2d619f9.png">

Reply:
Fixed in bc2e794db1918a674f997688af1d8dfc81f4783f